build/blocks/block-coauthor-description/block.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 2,
+  "name": "cap/coauthor-description",
+  "version": "0.1.0",
+  "title": "Co-Author Description",
+  "category": "widgets",
+  "icon": "smiley",
+  "description": "Display a co-author's description.",
+  "supports": {
+    "html": false,
+    "color": {
+      "link": true,
+      "text": true,
+      "background": true,
+      "__experimentalDefaultControls": {}
+    },
+    "typography": {
+      "fontSize": true,
+      "lineHeight": true,
+      "__experimentalFontFamily": true,
```

Comment:
They are in WordPress Core. I was careful to only use things that are generally available. There are other things labeled __unstable* which I stayed away from.

My understanding is that WordPress does not want any more __experimental things to be added because they never go away. I am basing that on this link.

https://developer.wordpress.org/block-editor/contributors/code/coding-guidelines/#legacy-experimental-apis-plugin-only-apis-and-private-apis